.github/workflows/tests.yml

```diff
@@ -0,0 +1,185 @@
+name: tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  linux:
+
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        php: ["5.6", "7.0", "7.1", "7.2", "7.3", "7.4"]
+        stability: ["Stable"]
+        include:
+          - php: "5.6"
+            stability: "Lowest"
+          - php: "8.0"
+            stability: "Ignore"
+
+    name: PHP ${{ matrix.php }} - ${{ matrix.stability }} - Linux
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          tools: composer:snapshot
```

Comment:
What is `composer:snapshot` for? Why can't we use the latest stable Composer which cames by default?

Reply:
Whoops, that can be changed to just `composer` now.